tests/NCronJob.Tests/Helpers/TestHelper.cs

```diff
@@ -192,20 +245,21 @@ private async IAsyncEnumerable<object> GetCompletionJobsAsync(int expectedJobCou
     }
 }
 
-public sealed class Storage
+public sealed class Storage(TimeProvider timeProvider)
 {
 #if NET9_0_OR_GREATER
     private readonly Lock locker = new();
 #else
     private readonly object locker = new();
 #endif
-    public IList<string> Entries { get; private set; } = [];
+    public IList<string> Entries => new ReadOnlyCollection<string>(TimedEntries.Select(e => e.Item2).ToList());
```

Comment:
Wrapped into a ReadOnlyCollection to prevent direct altering operations against the `Entries` property (`storage.Entries.Add()`, `storage.Entries.Clear()`, ...)